web_tree_image/static/src/js/web_tree_image.js

```diff
@@ -2,6 +2,8 @@
     OpenERP, Open Source Management Solution
     This module copyright (C) 2014 Therp BV (<http://therp.nl>)
                           (C) 2013 Marcel van der Boom <marcel@hsdev.com>
+    Copyright (C) 2011-Today Serpent Consulting Services Pvt. Ltd.
```

Comment:
Can you replace *2011-Today* by *2016*? See https://www.gnu.org/prep/maintain/html_node/Copyright-Notices.html